### PR TITLE
Add hack/verify-unittest-existence.sh

### PR DESCRIPTION
**Pull request by oomichi**
**What type of PR is this?**
/kind cleanup

**What this PR does / why we need it**:

To improve unit test coverage, this tool lets us know which modules don't have corresponding unit test modules.

ref: https://github.com/kubernetes/test-infra/issues/5231

### CODE REVIEW

File: test/cmd/check-unittest-existence.sh [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# This checks existence of corresponding unit test for go modules.
+
+get_list_of_unntested_modules() {
```
Comment (atoato88) [inline test/cmd/check-unittest-existence.sh]: replace "unntested" with "untested"?

Comment (oomichi) [reply test/cmd/check-unittest-existence.sh]: Thanks!

**Comment by oomichi** [comment]
Maybe it is useful for using this as hack/verify-foo.sh instead

**Comment by spiffxp** [comment]
/priority important-longterm
/retest
... not really sure why this isn't triggering tests?

**Pull request closed without merging**